ddmore.Rmd

```diff
@@ -20,10 +20,23 @@ data_dir <- "./testdata/nmoutputs"
 file_paths <- list.files(data_dir, "\\.lst$", full.names = TRUE)
 bdf <- function(.filename){
   df <- data.frame( stringsAsFactors = FALSE,
-                    filename = as.character(basename(.filename)),  
-                    lines = readr::read_lines(.filename)) 
+                    file_name = as.character(basename(.filename)),  
+                    line_text = readr::read_lines(.filename))  %>%    
+                        mutate (line_number = row_number()) %>% 
+                          select(file_name, line_number, line_text)
 }
-df <- map_df(file_paths[1:2], bdf)
+df1 <- map_df(file_paths, bdf)
+df <- df1 %>% 
+        mutate(has_dollar_est = grepl(x = df$line_text, pattern = "$EST", fixed = TRUE, useBytes = TRUE)) %>% 
```

Comment:
this will still match like $ESTIMATE right? I believe so, just confirming

Reply:
yes